app/models/comment.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: comments
+#
+#  id         :bigint           not null, primary key
+#  user_id    :bigint           not null
+#  post_id    :bigint           not null
+#  text       :text
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post, foreign_key: :post_id
+
+  validates :text, presence: true
+
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#update_posts_comments_counter` method every time a new `comment` has been saved to the database, then make the `update_posts_comments_counter` method private

Reply:
Implemented the change. Thanks again @DeliceLydia